docs/usage/self-hosted-experimental.md

```diff
@@ -129,7 +108,3 @@ Suppress the pre-commit support warning in PR bodies.
 ## `RENOVATE_X_USE_OPENPGP`
```

Comment:
Any reason why this is migrated?

Reply:
We will need to pass it via the global config directly, instead of accessing it using GlobalConfig class

Reply:
Did you mean why it is not converted to a experimental flag?